docs/page-markup.md

```diff
@@ -51,5 +51,38 @@ Thus, notice that:
 
 ## Microdata markup
 
-Microdata format support is coming soon.
+SWG uses schema.org markup. Using Microdata, the mark up could look like this:
 
+```
+<div itemscope itemtype="http://schema.org/NewsArticle http://schema.org/Other" id="top">
+  <meta itemprop="isAccessibleForFree" content="false"/>
+  <div itemprop="isPartOf" itemscope itemtype="http://schema.org/CreativeWork http://schema.org/Product">
+    <meta itemprop="name" content="New York Times"/>
+    <meta itemprop="productID" content="pub1:premium"/>
+  </div>
+  <div itemprop="articleBody" class="paywalled-section">
+    Paid content possibly.
+  </div>
+  <div itemscope itemtype="http://schema.org/OtherSection" id="bottom">
+    <meta itemprop="isAccessibleForFree" content="True"/>
+  </div>
+</div>
+```
+
+The requirements to retreive a usable configuration from embedded microdata are:
+ 1. Presence of atleast one div marked up as itemscope with itemtype containing 'NewsArticle' in
+    schema vocabulary (indicated by URL http://schema.org/)
+ 2. With presence of such a node as parent, a valid configuration is returned if it
```

Comment:
We should reduce as people familiar with microdata will know what this means. E.g. "NewsArticle can specify productID within a sub-itemtype of Product, and isAccessibleForFree". The details of validation shouldn't be in the docs.

Reply:
Done